components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIAndAPIProductCommonUtil.java

```diff
@@ -639,4 +647,74 @@ public static void addClientCertificates(String pathToArchive, APIProvider apiPr
             throw new APIImportExportException(errorMessage, e);
         }
     }
+
+    /**
+     * Get the subscription level policy names of an API/API Product
+     *
+     * @param apiTypeWrapper API or API Product to be exported
+     * @throws APIImportExportException
```

Comment:
No exception is thrown in this method. Please correct the java docs.

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/commit/eb9b7181c0f6cb5b9624602d91d28a7479bf551b